test/recur_expansion_test.js

```diff
@@ -37,8 +37,8 @@ suite('recur_expansion', function() {
   suite('initialization', function() {
     test('successful', function() {
       assert.deepEqual(
-        new Date(2012, 9, 2, 10),
```

Comment:
It was necessary to modify these tests due to previous behavior not accounting for time zones. When calling the `Date()` ctor with numerical parameters, those parameters are interpreted as being in the user's time zone; the tests passed previously because the time zone definition in the files was not used and none of the tests in this file registered the `America/Los_Angeles` time zone with the time zone service.